docs/Installation/install-on-linux.rst

```diff
@@ -0,0 +1,75 @@
+****************************
+Building Tahoe-LAFS on Linux
+****************************
+
+Tahoe-LAFS has made packages available for installing on many linux and BSD distributions. Debian and Ubuntu users can use ``apt-get install tahoe-lafs``. If you are working on a Linux distribution which does not have Tahoe-LAFS or are looking to hack on the source code, you can build Tahoe-LAFS yourself:
+
+Prerequisites
+=============
+
+Make sure the following are installed:
+
+* **Python 3's latest version**: Check for the version by running ``python --version``.
+* **pip**: Most python installations already include ``pip``. However, if your installation does not, see `pip installation <https://pip.pypa.io/en/stable/installing/>`_.
+* **virtualenv**: Use ``pip`` to install virtualenv:
+  ::
+   pip install --user virtualenv
+* **C compiler and libraries**:
+
+    * ``python-dev``: Python development headers.
+    * ``libffi-dev``: Foreign Functions Interface library.
+    * ``libssl-dev``: SSL library, Tahoe-LAFS needs OpenSSL version 1.1.1c or greater.
+    .. note::
+       If you are working on Debian or Ubuntu, you can install the necessary libraries using ``apt-get``:
+       ::
+        apt-get install python-dev libffi-dev libssl-dev 
+       On an RPM-based system such as Fedora, you can install the necessary libraries using ``yum`` or ``rpm``. However, the packages may be named differently.
+
+Install the Latest Tahoe-LAFS Release
+=====================================
+
+If you are looking to hack on the source code or run pre-release code, we recommend you create a ``virtualenv`` instance and install Tahoe-LAFS from a source tarball into that:
```

Comment:
I think this advise should be more-general: everyone installing using `pip` should be installing into a `virtualenv` (no matter their motivation). So something like, "To install Tahoe-LAFS directly from source, we recommend you create a `virtualenv` ..."

Reply:
I took out the "tarball" reference in my suggestion since nothing here actually downloads a `.tar.gz` --- but that said, users who do not want to hack on Tahoe don't need to do a `git clone` at all and can tell `pip` to install from a tarball directly -- and GitHub can produce `zip` archives automagically for repos. For example, `pip install https://github.com/tahoe-lafs/tahoe-lafs/archive/refs/heads/master.zip`